packages/server/src/api/routes/tests/application.spec.ts

```diff
@@ -1,4 +1,7 @@
 import { DEFAULT_TABLES } from "../../../db/defaultData/datasource_bb_default"
+import { setEnv } from "../../../environment"
+
+setEnv({ USE_LOCAL_COMPONENT_LIBS: "0" })
```

Comment:
nit: to avoid changing global state in this test, I'd recommend putting this in `beforeAll`, assigning the result to `envCleanup: () => void`, and then calling it in `afterAll`.

Reply:
Also if this is what triggers MinIO to get used, I'd put a comment above it explaining that.

Reply:
Jest don't allow changing global state, so it should not be neede

Reply:
Done!